app/views/layouts/application.html.erb

```diff
@@ -78,10 +78,40 @@
   <link href='https://api.mapbox.com/mapbox.js/plugins/leaflet-fullscreen/v1.0.1/leaflet.fullscreen.css' rel='stylesheet' />
   <script src="https://cdn.jsdelivr.net/npm/leaflet-search@2.4.0/dist/leaflet-search.min.js"></script>
   <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/leaflet-search@2.4.0/dist/leaflet-search.min.css">
+<link rel="stylesheet" type="text/css" href="app/assets/stylesheets/application.css">
   </head>
 
-  <body>
-
+<div id="loader-wrapper">
```

Comment:
Also, i'd like to suggest this get separated out into a partial template. Have you seen how this is done, or would you need more support to do it? Thanks!

Reply:
Oh.. I'm so sorry @jywarren .. Didn't take a look here.. Will do the changes required now